packages/settingregistry/src/tokens.ts

```diff
@@ -17,6 +17,14 @@ import { ISignal } from '@lumino/signaling';
 import { ISchemaValidator } from './settingregistry';
 import type { RJSFSchema, UiSchema } from '@rjsf/utils';
 
+/**
+ * A service to connect to the server endpoint.
```

Comment:
Did you mean

```suggestion
 * A service to connect to the settings endpoint.
```

or was it deliberate?

Reply:
Right, `settings` makes more sense.